packages/firebase_auth/lib/firebase_auth.dart

```diff
@@ -215,6 +215,21 @@ class FirebaseAuth {
     return currentUser;
   }
 
+  Future<FirebaseUser> signInWithTwitter({
```

Comment:
We should add Dartdoc for this (along with the other methods if you're feeling ambitious)